### DISCUSSION TITLE
How to turn off matrix in shutdown mode?

### DISCUSSION THREAD

**Discussion by hitmefast** [seerge/g-helper#910] [General]
Well basicly how to turn off matrix in shutdown mode? As soon as I close the laptop it starts matrix in starfield and I seems can't find how to turn that off or how to even modify it.

**Comment by seerge** [comment]
@hitmefast it's turned on on hardware level, if matrix itself is ON, you can turn matrix completely off - then it will shutdown animation won't play neither

**Reply by hitmefast** [reply]
@seerge I know that part, but if I pick to have any matrix and then shutdown or close the laptop it turns it off but then starts the stardust minutes after and won't stop no matter the time. But what is the point of closing your pc if it will drain the battery no matter what or will do that even if you leave home with some stardust that no body cares or can see.

**Reply by seerge** [reply]
Select disable matrix on battery
